mongodb/map.jinja

```diff
@@ -1,14 +1,19 @@
 {% set mongodb = salt['grains.filter_by']({
     'default': {
-        'pkg': 'mongodb',
-        'service': 'mongodb',
-        'conf_file': '/etc/mongodb/mongodb.conf',
-        'config': {
-            'key': 'value',
-        },
+      'key': '7F0CEB10',
+      'keyserver': 'keyserver.ubuntu.com',
+      'pkgs': [
+        {'mongodb-10gen': '*'},
+      ],
+      'cluster_key_file': '/etc/mongodb_key',
+      'bind_ip': '127.0.0.1',
+      'port': 12017,
+      'enable_journal': True,
```

Comment:
Not with the way it's currently being used.

```
{% if mongodb.enable_journal %}
journal = true
{% else %}
journal = false
nojournal = true
{% endif %}
```

I also wrote a note in the comment addressing why I chose to keep the conditional instead of the more declarative alternative.